tests/js/spec/actionCreators/__snapshots__/health.spec.jsx.snap

```diff
@@ -0,0 +1,41 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Health ActionCreator requests timeseries 1`] = `
+Object {
+  "includePrevious": undefined,
```

Comment:
What is includePrevious exactly (isn't that clear from the name)? Would it be better represented as a boolean?